docs/readmes/orc8r/dev_minikube.md

```diff
@@ -6,120 +6,117 @@ hide_title: true
 
-```
-helm template orc8r charts/secrets \
-    --namespace magma \
-    --set-string secret.certs.enabled=true \
-    --set-file secret.certs.files."rootCA\.pem"=charts/secrets/.secrets/certs/rootCA.pem \
-    --set-file secret.certs.files."bootstrapper\.key"=charts/secrets/.secrets/certs/bootstrapper.key \
-    --set-file secret.certs.files."controller\.crt"=charts/secrets/.secrets/certs/controller.crt \
-    --set-file secret.certs.files."controller\.key"=charts/secrets/.secrets/certs/controller.key \
-    --set-file secret.certs.files."admin_operator\.pem"=charts/secrets/.secrets/certs/admin_operator.pem \
-    --set-file secret.certs.files."admin_operator\.key\.pem"=charts/secrets/.secrets/certs/admin_operator.key.pem \
-    --set-file secret.certs.files."certifier\.pem"=charts/secrets/.secrets/certs/certifier.pem \
-    --set-file secret.certs.files."certifier\.key"=charts/secrets/.secrets/certs/certifier.key \
-    --set-file secret.certs.files."nms_nginx\.pem"=charts/secrets/.secrets/certs/nms_nginx.pem \
-    --set-file secret.certs.files."nms_nginx\.key\.pem"=charts/secrets/.secrets/certs/nms_nginx.key \
-    --set=docker.registry=$DOCKER_REGISTRY \
-    --set=docker.username=$DOCKER_USERNAME \
-    --set=docker.password=$DOCKER_PASSWORD |
-    kubectl apply -f -
-```
+A minimal values file is at `${MAGMA_ROOT}/orc8r/cloud/helm/orc8r/examples/minikube_values.yml`
 
-### Create Values File
+- Copy that file to `${MAGMA_ROOT}/orc8r/cloud/helm/orc8r.values.yaml`
+- Replace `IMAGE_REGISTRY_URL` with your registry and `IMAGE_TAG` with your tag
+- Make additional edits as desired
 
-A minimum values file that can be used to deploy orc8r is at `${MAGMA_ROOT}/orc8r/cloud/helm/orc8r/examples/minikube_values.yml`.
-Use that file and make sure to replace `<DOCKER_REGISTRY>` with your registry and `<TAG>` with your tag.
 
-Save this file wherever you want.
+### Install charts
 
-### Install with Helm
+This section describes how to install based on local charts. However, you can also install charts from the official chart repositories
+- Stable: https://artifactory.magmacore.org/artifactory/helm/
+- Test: https://artifactory.magmacore.org/artifactory/helm-test/
 
-```
-$ cd ${MAGMA_ROOT}/orc8r/cloud/helm/orc8r
-$ helm dep update
-$ helm install orc8r --namespace magma . --values=<path-to-values-file>
+Install base `orc8r` chart
+
+```bash
+cd ${MAGMA_ROOT}/orc8r/cloud/helm/orc8r
+helm dep update
+helm upgrade --install --namespace orc8r --values ${MAGMA_ROOT}/orc8r/cloud/helm/orc8r.values.yaml orc8r .
 ```
 
-If everything worked you should get something like this:
+Optionally install other charts, like the `lte` charts, similarly
 
+```bash
+cd ${MAGMA_ROOT}/lte/cloud/helm/lte-orc8r
+helm dep update
+helm upgrade --install --namespace orc8r --values ${MAGMA_ROOT}/orc8r/cloud/helm/lte.values.yaml lte .
 ```
-$ kubectl -n magma get pods
+
+If successful, you should get something like this
+
+```bash
+$ kubectl --namespace orc8r get pods
```

Comment:
Do we want `$` here? Looks like we don't have it anywhere else now. (Or if you want to keep it to differentiate between the command and output that is OK too.)

Reply:
`$` is included when the code block also shows the output of the cmd -- to differentiate cmd vs output